doc/development_process/issues.rst

```diff
@@ -9,3 +9,54 @@ a reference to the feature by mentioning the corresponding Issue or pull-request
 
 At any time it should be possible to establish the origin of a change and the
 reason behind it by following the references in the code.
+
+
+Feature Tracking
+*****************
+For feature tracking we use Github labels to classify new features and
+enhancements. The following is the description of each category:
+
+Enhancement
+  Changes to existing features that are not considered a bug and would not a
+  block a release. This is an incremental enhancement to a feature that already
+  exists in Zephyr.
+
+Feature request
+  A request for a feature that is not on the roadmap yet that has not been
+  vetted yet and needs further discussion and details.
+
+Feature
+  A high level roadmap feature with a detailed proposal with an owner. Features
+  require a detailed architectural design and must go through an RFC process.
+
+The following workflow should be used to process features and roadmap items.
+
+Requesting new features is done using the Github issues system using the label
+``feature-request``. This is the formal way for asking for a new feature in
+Zephyr and a way to signal to the project interest in the feature and a
+readiness and willingness to implement the feature and deliver it in one of the
+upcoming releases. The creator of the request should assign the item to self if
+they wish to driver the implementation and contribute it to Zephyr. A feature
+request can have a companion RFC with more details on the feature and the
+proposed implementation.
+
+- Label new features requests as ``feature-request``
+- Project discusses new ``feature-request`` items regulary and triages them.
+  Items are examined for similarity with existing features, how they fit with
+  the project goals and other timeline considerations. The priority is
+  determined as follows:
+
+  - High = Next milestone
+  - Medium = As soon as possible
+  - Low = Best effort
+
+- After the initial discussion and triaging, the label is moved from
+  ``feature-request`` to ``feature`` with the target milestone and an assignee.
+
+All items marked as ``feature-request`` are non-binding and are open for grabs,
+meaning that they can be picked up and implemented by any project member or the
+community.
+
+Items labeled as ``features`` are roadmap items, short and long term that shall
```

Comment:
When you say that something labeled as feature, it must be a roadmap item, you mean that they must be listed in the GH roadmap project directly? or can they be indirectly linked? (thru a "master" issue)